smoke/features/data/rbac.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
```
$ shellcheck rbac.sh 

In rbac.sh line 22:
if [ ${1} == "sharedconfigmap" ]; then
     ^--^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
if [ "${1}" == "sharedconfigmap" ]; then


In rbac.sh line 31:
oc create rolebinding shared-resource-my-shared --role=shared-resource-my-shared --serviceaccount=$(oc project -q):default
                                                                                                  ^--------------^ SC2046: Quote this to prevent word splitting.

For more information:
  https://www.shellcheck.net/wiki/SC2046 -- Quote this to prevent word splitt...
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```